### ISSUE TITLE
Tray icon not visible with KDE

### ISSUE THREAD

**Issue by jlippuner** [keepassxreboot/keepassxc#4316]
I just recently switched to KeePassXC (from Enpass) and I love it!

I'm using KDE with the dark theme and I'm having some issues with the tray icon. Without changing anything, KeePassXC will show up with the dark theme (although the icons in the menu bar are somewhat hard to see), but the tray icon is completely invisible. It is actually in the system tray and I can click on it, but it just shows up as a blank space between the other icons.

Following the suggestion I found here (https://www.reddit.com/r/gnome/comments/9a8p24/keepassxc_icon_is_not_showing_up_in_gnomeshell/), I tried to run KeePassXC by first setting `XDG_CURRENT_DESKTOP=Gnome`. With this, the system tray icon shows up, but the KeePassXC GUI is now not using the dark theme but the regular light theme.

It would be great to make the system tray icon work with KDE.

**Comment by droidmonkey** [comment]
Screenshot please

**Comment by jlippuner** [comment]
Sorry for the delay. Here it is with setting `XDG_CURRENT_DESKTOP=Gnome` (everything looks great, except that it's not using the dark theme):
<img width="685" alt="keepassxc_with_XDG_CURRENT_DESKTOP" src="https://user-images.githubusercontent.com/2790769/74387571-81801000-4dbe-11ea-92cf-13d0de504dc5.png">

And here it is without setting the `XDG_CURRENT_DESKTOP` environment variable explicitly when launching `keepassxc` (so it picks up the default value, which is `KDE`). Note how the grayed out tool icons are not visible and the system tray icon is not visible, although it is clickable and that works.
<img width="685" alt="keepassxc_without_XDG_CURRENT_DESKTOP" src="https://user-images.githubusercontent.com/2790769/74387612-9f4d7500-4dbe-11ea-99e2-8b1cdf84b5ce.png">

**Comment by droidmonkey** [comment]
Oh you are running the develop branch...

**Comment by phoerious** [comment]
The tray icon is weird, the application icons are a known and expected issue that will be fixed once my redesign PR is merged.

**Comment by jlippuner** [comment]
> Oh you are running the develop branch... This might not be an issue after more PR's are merged.

I did compile it myself from the git repo, but I used the build-release script to set version 2.5.3, so shouldn't I be running version 2.5.3 then?

**Comment by Hyperboid** [comment]
On release 2.7.6 via flathub, exact same behavior. `XDG_CURRENT_DESKTOP=Gnome` fixes the issue, but shows the default Qt theme

Fedora 40 Branched, Breeze theme, light color scheme.

Reopen, or new issue?

**Comment by eleksis** [comment]
Issue still exists (tray icon invisible) with:
KeepassXC 2.7.8 (via snap)
KDE  Plasma 5.27.5 (Breeze dark theme)

@droidmonkey can you re-open please?

**Comment by droidmonkey** [comment]
That's not on us, we provide an icon to qt which then works with your desktop env to show it.

Also this issue is 5 years old and totally irrelevant today.

**Comment by eleksis** [comment]
Ah, looks like this is a snap(+KDE?) issue.

**Comment by SevenChalices** [comment]
Debian Trixie with KDE and KeePassXC installed as keepassxc-full (from official repository and with apt, no flatpak/snap/appimage/etc.. Everything works fine, also with the browser plugin for it (for Firefox in my case), but in my systray KeePassXC is not even listed - that was never the case. And I just reinstalled my system few days ago without any real changes to OS, neither to software(-configs) while just about one week ago everything worked fine together.

(You see that the browser plugin for KeePassXC without it's minimize-to-tray feature is basically worthless...)
